clang/test/SemaSYCL/attr-add-ir-attributes-packs.cpp

```diff
@@ -0,0 +1,2196 @@
+// RUN: %clang_cc1 -fsycl-is-device -std=gnu++11 -fsyntax-only -verify %s
```

Comment:
I believe we plan to eventually split AST and diagnostic tests into separate files. So we may want to separate these into 2 files. @AaronBallman can you confirm?

Personally I prefer it separate. Its a lot more readable.

Reply:
I prefer separate as well, FWIW. AST dump tests typically live in the `clang/test/AST` directory, and splitting makes it easier to test diagnostics that produce an error without also impacting the AST dumping tests.

Reply:
The tests have been split into AST dump tests and sema tests.